plugin-server/src/utils/db/db.ts

```diff
@@ -154,28 +154,31 @@ export class DB {
     // Postgres
 
     public postgresQuery<R extends QueryResultRow = any, I extends any[] = any[]>(
-        queryTextOrConfig: string | QueryConfig<I>,
+        queryString: string,
         values: I | undefined,
         tag: string,
         client?: PoolClient
     ): Promise<QueryResult<R>> {
         return instrumentQuery(this.statsd, 'query.postgres', tag, async () => {
             let fullQuery = ''
-            if (typeof queryTextOrConfig === 'string') {
+            if (typeof queryString === 'string') {
```

Comment:
Note: Purposefully not touching this - not sure why this even exists.

Reply:
get rid of this check - was needed because of the typing in the function, but you just changed it.

Reply:
Well the whole variable is also dead